fdb-extensions/src/test/java/com/apple/foundationdb/tuple/ByteArrayUtil2Test.java

```diff
@@ -57,12 +65,40 @@ public void testLoggable() throws Exception {
     }
 
     @Test
-    public void testUnprint() throws Exception {
+    void testUnprint() {
         byte[] allBytes = new byte[Math.abs((int)Byte.MIN_VALUE) + Byte.MAX_VALUE];
         for (byte b = Byte.MIN_VALUE; b < Byte.MAX_VALUE; b++) {
             allBytes[b - Byte.MIN_VALUE] = b;
         }
         assertArrayEquals(allBytes, ByteArrayUtil2.unprint(ByteArrayUtil2.loggable(allBytes)));
         assertArrayEquals(allBytes, ByteArrayUtil2.unprint(ByteArrayUtil.printable(allBytes)));
     }
+
+    @Nonnull
+    static Stream<Long> testRandomBytes() {
+        return RandomizedTestUtils.randomSeeds(0x0fdbL, 0x5ca1eL, 123456L, 78910L, 1123581321345589L);
```

Comment:
It's been a while, so I don't totally remember. I think the non-hex value might have come from running this with random seends enabled but the asserts modified so that I could see whether it created a different string with `printable` or `loggable`